src/utils/utils.service.js

```diff
@@ -59,6 +59,47 @@ class UtilsService {
       return index
     }, {})
   }
+
+  checkAndResolveConflicts (changedDoc, pouchdb) {
+    let preferredRevision = {}
+    if (!changedDoc._conflicts) {
+      return
+    }
+
+    if (!changedDoc.updatedAt) {
+      preferredRevision = changedDoc
+      changedDoc._conflicts.map((conflictingRev) => {
+        if (conflictingRev.updatedAt) {
```

Comment:
Shouldn't `updateAt` be checked to see if it's newer than the currently chosen `preferredRevision`?